scss/foundation-styles/buttons.scss

```diff
@@ -0,0 +1,126 @@
+.btn {
+  font-family: "Nunito Sans";
+  font-weight: $btn-font-weight;
+  border-radius: 0;
```

Comment:
If we wanna get rid of `border-radius` for buttons we should do it by removing `$btn-border-radius` in the codebase instead of introducing another override. @natalieworth just to confirm - do we want to reset all buttons's border-radius so they don't have rounded edges?

Reply:
@mmmavis I went ahead and removed this override and instead updated the value of `$btn-border-radius` to `0px`. Removing that variable would involve some variable + mixin clean up which I think are beyond the scope of this small ticket and can be done separately.

Reply:
Sounds good. We can clean those up as follow up. Can you create a ticket for this as a reminder?